ansible/roles/letsencrypt/vars/main.yml

```diff
@@ -0,0 +1,28 @@
+letsencrypt_docker_image: quay.io/letsencrypt/letsencrypt
+letsencrypt_docker_name: certbot
+letsencrypt_admin_email: admin@openobservatory.org
+letsencrypt_workdir: /var/lib/letsencrypt
+letsencrypt_port: 80
+letsecnrypt_pre_hook: docker stop {{ app_env }}
```

Comment:
I though you meant `app_env`. 
You are right though technically it's OK since the variable has the same name (consistent typo).
Thank you for spotting this!